locale/zh-cn/site.json

```
@@ -80,7 +80,7 @@
       "verify-text": "如何校验"
     },
     "install-on-linux": {
-      "text": "Installing Node.js via binary archive on Linux"
+      "text": "在Linux上，通过二进制文件安装Node.js"
```

```suggestion
      "text": "在 Linux 上，通过二进制文件安装 Node.js"
```

中英文之间加下空格比较好吧，与上面的保持一致

---

thanks